sdk/core/core-auth/review/core-auth.api.md

```diff
@@ -19,6 +19,13 @@ export class AzureKeyCredential implements KeyCredential {
     update(newKey: string): void;
 }
 
+// @public
+export class AzureSasCredential implements SasCredential {
+    constructor(signature: string);
```

Comment:
Do we want to be more explicit with the name here and go with `sharedAccessSignature`? It would be more searchable than `signature`

Reply:
I believe that this was discussed in the arch board review and `Sas` was chosen for the type name, and documentation is asked to use the full `shared access signature` rather than abbreviating.    

@tg-msft:  Can you please confirm my understanding?

Reply:
After looking at the other languages, they also used `signature` here instead of something more verbose.  I did ensure that the associated documentation now says "shared access signature" in prose when talking about these members instead of just "signature".  I feel like that's a reasonable middle ground.